<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternVisualizer</class>
 <widget class="QWidget" name="PatternVisualizer">
  <property name="windowTitle">
   <string>Pattern Visualizer</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="mPatternVisLayout">
   <item row="0" column="0">
    <widget class="PatternVisualizer::PatternsTreeView" name="patternsTree">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="PatternVisualizer::GlobalPlotSettings" name="globalPlotSettingsWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="PatternVisualizer::PolarConstantAzElPlotSettings" name="polarConstantAzimuthPlotSettings" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="PatternVisualizer::PolarConstantAzElPlotSettings" name="polarConstantElevationPlotSettings" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="PatternVisualizer::Generic3DPlotSettings" name="generic3DPlotSettings" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PatternVisualizer::PatternsTreeView</class>
   <extends>QTreeView</extends>
   <header>PatternsTreeView.hpp</header>
  </customwidget>
  <customwidget>
   <class>PatternVisualizer::Generic3DPlotSettings</class>
   <extends>QWidget</extends>
   <header>Generic3DPlotSettings.hpp</header>
  </customwidget>
  <customwidget>
   <class>PatternVisualizer::GlobalPlotSettings</class>
   <extends>QWidget</extends>
   <header>GlobalPlotSettings.hpp</header>
  </customwidget>
  <customwidget>
   <class>PatternVisualizer::PolarConstantAzElPlotSettings</class>
   <extends>QWidget</extends>
   <header>PolarConstantAzElPlotSettings.hpp</header>
  </customwidget>
 </customwidgets>
</ui>
